src/ansys/mapdl/core/mapdl_core.py

```diff
@@ -286,7 +286,7 @@ def __init__(
         self.check_parameter_names = start_parm.get("check_parameter_names", True)
 
         # Setting up loggers
-        self._log: logging.Logger = logger.add_instance_logger(
+        self._log: logger = logger.add_instance_logger(
```

Comment:
Unless it is done on purpose, the ``logger`` variable will be in conflict with ``from ansys.mapdl.core import LOG as logger``. You might need to change the variable name or the import alias.

Reply:
Here the ``logger`` is an annotation. :)